R/pkg/R/RDD.R

```diff
@@ -85,7 +85,7 @@ setMethod("initialize", "PipelinedRDD", function(.Object, prev, func, jrdd_val)
 
   isPipelinable <- function(rdd) {
     e <- rdd@env
-    !(e$isCached || e$isCheckpointed)
+    ! (e$isCached || e$isCheckpointed)
```

Comment:
lets just create a variable here `cachedOrCheckpointed <- e$isCached || e$isCheckpointed` and return `!cachedOrCheckpointed` here